spec/models/allyship_spec.rb

```diff
@@ -83,41 +77,21 @@
   end
 
   context 'when destroying' do
-    describe 'deletes pertinent allyship' do
-      let!(:user) { create(:user1) }
-      let!(:ally) { create(:user2) }
+    let!(:user) { create(:user1) }
+    let!(:ally) { create(:user2) }
+    let(:allyship) { Allyship.where(user_id: user.id, ally_id: ally.id)[0] }
 
-      before do
-        Allyship.create(user_id: user.id,
-                        ally_id: ally.id,
-                        status: 'accepted')
-        Allyship.create(user_id: ally.id,
-                        ally_id: user.id,
-                        status: 'accepted')
-      end
+    before do
+      Allyship.create(user_id: user.id, ally_id: ally.id, status: 'accepted')
+      Allyship.create(user_id: ally.id, ally_id: user.id, status: 'accepted')
+    end
 
-      it 'deletes the allyship' do
-        allyship_expected = Allyship.where(user_id: user.id,
-                                           ally_id: ally.id)[0]
-        expect { allyship_expected.destroy }
-          .to change { Allyship.count }
-      end
+    it 'deletes the allyship' do
+      expect { allyship.destroy }.to change { Allyship.count }
```

Comment:
Can we add a `by` condition to ensure the count is changing by the ammount we expect?

Example:

```
let(:subject) { allyship.destroy }

it { expect(subject).to change(Allyship, :count).by(1) }
```